project-base/tests/App/Acceptance/acceptance/PageObject/Front/OrderPage.php

```diff
@@ -100,9 +104,14 @@ public function assertFirstNameIsFilled($firstName)
     public function fillPersonalInfo($firstName, $lastName, $email, $telephone)
     {
         $this->fillFirstName($firstName);
-        $this->tester->fillFieldByName('order_personal_info_form[lastName]', $lastName);
-        $this->tester->fillFieldByName('order_personal_info_form[email]', $email);
-        $this->tester->fillFieldByName('order_personal_info_form[telephone]', $telephone);
+
+        $lastNameInput = $this->webDriver->findElement(WebDriverBy::name('order_personal_info_form[lastName]'));
```

Comment:
Yup, now I'm almost convinced we need `findElementByName` method 😄

Reply:
Resolved different way in StrictWebDriver